psl-core/src/main/java/org/linqs/psl/application/inference/online/OnlineServer.java

```diff
@@ -53,13 +60,17 @@
     private Set<ClientConnectionThread> clientConnectionThreads;
     private BlockingQueue<OnlineMessage> queue;
     private ConcurrentMap<UUID, ClientConnectionThread> messageIDConnectionMap;
+    private List<Rule> rules;
+    private File tmpFile;
```

Comment:
When used as an identifier always prefer "temp" to "tmp".
(Since "tmp" is a very specific POSIX location.")